bigtable/conformance_test.sh

```diff
@@ -49,11 +48,12 @@ function cleanup() {
 }
 trap cleanup EXIT
 
-pushd $conformanceTestsHome > /dev/null;
-    # Run the conformance tests
-    go test -v -proxy_addr=:$testProxyPort | tee -a $sponge_log
-    RETURN_CODE=$?
-popd > /dev/null;
+# Run the conformance tests
+cd $conformanceTestsHome
+go install golang.org/dl/go1.20.2@latest
+go1.20.2 download
+go1.20.2 test -v -proxy_addr=:$testProxyPort | tee -a $sponge_log
```

Comment:
Should this be done with the golang version that is currently selected in the strategy matrix?

Reply:
No.
This is running the conformance tests. Conformance tests can only be run on Go 1.20.2

Strategy matrix is for testproxy versions. https://github.com/googleapis/google-cloud-go/blob/main/README.md?plain=1#L31-L36